src/user-booking/user-booking.service.ts

```diff
@@ -0,0 +1,31 @@
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+
+
+import {Observable} from 'rxjs';
+import {Reservation} from '../models/reservation';
+import {AppApiConst} from "../app/app.api.const";
+import {tap} from "rxjs/operators";
+
+@Injectable()
+export class UserBookingService {
+  constructor(private http: HttpClient) {
+  }
+
+  httpOptions = {
+    headers: new HttpHeaders({'Content-Type': 'application/json'})
+  };
+
+  getAllReservationUser(email: string): Observable<Reservation[]> {
+    const url = `${AppApiConst.USER_BOOKING}/${email}`;
```

Comment:
do we really need this const? why not to use it right in the eturn this.http.get<Reservation[]>(<URL here>) ?

Reply:
done